automation-packages/load-testing-okhttp/src/test/java/step/examples/loadtesting/okhttp/OkHttpKeywordTest.java

```diff
@@ -20,7 +20,10 @@ public void setUp() {
 
     @Test
     public void test() throws Exception {
-        ctx.run("OpenCart home", "{}");
```

Comment:
For the local run I wouldn't use the KeywordRunner but the new StepAutomationPackageRunner that runs the complete AP locally via JUnit. Simply create a class as follow under test/:

@RunWith(StepAutomationPackageRunner.class)
public class LocalAutomationPackageTest {
}